app/controllers/quotes_controller.rb

```diff
@@ -1,6 +1,13 @@
 class QuotesController < ApplicationController
   def new
     @quote = Quote.new
+    @lead = Lead.new
+    @service_expedition_time = [
```

Comment:
as this is a collection, variable name should be plural

Reply:
@service_expedition_options

Reply:
done